src/components/comparison/comparison-side/comparison-side.tsx

```diff
@@ -129,16 +125,16 @@ export const ComparisonSide = ({
   onViewStateChange,
   pointToTileset,
   onChangeLayers,
-  onInsertBaseMap,
-  onSelectBaseMap,
-  onDeleteBaseMap,
   onLoadingStateChange,
   onTilesetLoaded,
   onShowBookmarksChange,
   onAfterDeckGlRender,
   onInsertBookmarks,
   onUpdateSublayers,
 }: ComparisonSideProps) => {
+  const baseMaps = useAppSelector(selectBaseMaps);
+  const selectedBaseMapId = useAppSelector(selectSelectedBaseMapId);
+  const selectedBaseMap = baseMaps.find((map) => map.id === selectedBaseMapId);
```

Comment:
We could select necessary values in `DeckGlWrapper` as it already depens on Redux

Reply:
1. Removed props and selectors from comparison-side.tsx, debug-app.tsx, viewer-app.tsx:
  const baseMaps = useAppSelector(selectBaseMaps);
  const selectedBaseMapId = useAppSelector(selectSelectedBaseMapId);
  const selectedBaseMap = baseMaps.find((map) => map.id === selectedBaseMapId);
...
        showTerrain={selectedBaseMap?.id === "Terrain"}
        mapStyle={selectedBaseMap?.mapUrl}
2. Added selectors to the deck-gl-wrapper.tsx:
  const baseMaps = useAppSelector(selectBaseMaps);
  const selectedBaseMapId = useAppSelector(selectSelectedBaseMapId);
  const selectedBaseMap = baseMaps.find((map) => map.id === selectedBaseMapId);
  const showTerrain=selectedBaseMap?.id === "Terrain";
  const mapStyle=selectedBaseMap?.mapUrl;
3. Fixed 2 tests in the deck-gl-wrapper.spec.tsx